custom_components/openhasp/__init__.py

```diff
@@ -344,15 +344,15 @@ def __init__(self, hass, config, entry):
 
         self._subscriptions = []
 
-        with open(
-            pathlib.Path(__file__).parent.joinpath("pages_schema.json"), "r"
-        ) as schema_file:
-            self.json_schema = json.load(schema_file)
-
         self._attr_unique_id = entry.data[CONF_HWID]
         self._attr_name = entry.data[CONF_NAME]
         self._attr_icon = "mdi:gesture-tap-box"
 
+    def _read_file(self, path):
+        """Executor helper to read file."""
+        with open(path, "r") as src_file:
+            return src_file.read()
```

Comment:
Might this be the culprit for #136? Previously, the file handle returned by `open` was directly passed into `send_lines`. I presume, iterating over this handle yields lines, whereas `read` yields a string. And iterating over the string yields single characters. Maybe?

Reply:
don't think soo... it was and still is processed by `json.load()`